ocs_ci/pause/pause.py

```diff
@@ -0,0 +1,88 @@
+import argparse
+import logging
+import pickle
+
+import os
+
+from ocs_ci.framework import config
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.ocs import platform_nodes
+from ocs_ci.ocs.node import get_node_objs
+from ocs_ci.utility.retry import retry
+
+from ocs_ci.ocs.constants import (
+    NODE_OBJ_FILE, NODE_FILE, INSTANCE_FILE
+)
+
+
+def cycle_nodes(cluster_path, action):
```

Comment:
I think this function could use some logging statements.

Reply:
added some more logging